doc/examples/simple.yml

```diff
@@ -121,3 +121,9 @@ receivers:
     room_id: 85
     message_format: html
     notify: true
+- name: 'matrix'
+  matrix_configs:
+  - access_token: <access_token>
+    room_id: !<something>:matrix.org
```

Comment:
Is there a way to make this not tied to matrix.org homeserver? I would like to use this with my homeserver without having to federate a room to matrix.org